.eslintrc

```diff
@@ -1,3 +1,9 @@
 {
   "extends": "airbnb-base",
+  "env": {
+    "node": true
+  },
+  "rules": {
+    "no-console": "off"
```

Comment:
Id prefer not to add this, twitch.js is just a place holder for a quick second, hence the eslint disable.